RecoLocalMuon/RPCRecHit/test/testRPCPointProducer.py

```diff
@@ -1,49 +1,54 @@
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
-process.GlobalTag.globaltag = "START53_V27::All"
-
-process.load("Configuration.StandardSequences.Services_cff")
-process.load("Configuration.StandardSequences.GeometryDB_cff")
-process.load("Configuration.StandardSequences.Reconstruction_cff")
 process.load("Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff")
+process.GlobalTag.globaltag = "124X_dataRun3_Express_v5"
 
-process.load("FWCore.MessageService.MessageLogger_cfi")
 
-process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(10) )
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(2000) )
 
 process.source = cms.Source("PoolSource",
-    fileNames = cms.untracked.vstring(
-        #	'/store/data/Commissioning10/Cosmics/RECO/v3/000/127/155/005F9301-2E16-DF11-B60B-0030487CD6B4.root'
-        '/store/data/Run2012D/RPCMonitor/RAW/v1/000/207/900/1AFE2D3D-A836-E211-B86A-001D09F23D1D.root'
-        )                           
-)
+   fileNames = cms.untracked.vstring(
 
+# local copy for test only:
+'file:/eos/cms/store/group/dpg_rpc/comm_rpc/Sandbox/mileva/testRPCMon2022/1cb9263a-550d-4e86-92f3-f01574867137.root'
+)
+)
 
 process.load("RecoLocalMuon.RPCRecHit.rpcPointProducer_cff")
-process.rpcPointProducer.tracks = cms.InputTag("cosmicMuons") # for cosmicMuons
-
-process.out = cms.OutputModule("PoolOutputModule",
-  outputCommands = cms.untracked.vstring('drop *',
-        'keep *_dt4DSegments_*_*',
-        'keep *_cscSegments_*_*',
-        'keep *_rpcPointProducer_*_*',
-        'keep *_rpcRecHits_*_*',
-        'keep *_standAloneMuons_*_*',
-        'keep *_cosmicMuons_*_*',
-        'keep *_globalMuons_*_*'),
- fileName = cms.untracked.string('output.root')
+process.rpcPointProducer = cms.EDProducer('RPCPointProducer',
+  incldt = cms.bool(True),
+  inclcsc = cms.bool(True),
+  incltrack =  cms.bool(False),
+  debug = cms.untracked.bool(False),
+  rangestrips = cms.double(4.),
+  rangestripsRB4 = cms.double(4.),
+  MinCosAng = cms.double(0.85),
+  MaxD = cms.double(80.0),
+  MaxDrb4 = cms.double(150.0),
+  ExtrapolatedRegion = cms.double(0.6), # in stripl/2 in Y and stripw*nstrips/2 in X
+  cscSegments = cms.InputTag('dTandCSCSegmentsinTracks','SelectedCscSegments','OwnParticles'),
+  dt4DSegments = cms.InputTag('dTandCSCSegmentsinTracks','SelectedDtSegments','OwnParticles'),
+  tracks = cms.InputTag("standAloneMuons"),
+  TrackTransformer = cms.PSet(
+      DoPredictionsOnly = cms.bool(False),
+      Fitter = cms.string('KFFitterForRefitInsideOut'),
+      TrackerRecHitBuilder = cms.string('WithTrackAngle'),
+      Smoother = cms.string('KFSmootherForRefitInsideOut'),
+      MuonRecHitBuilder = cms.string('MuonRecHitBuilder'),
+      RefitDirection = cms.string('alongMomentum'),
+      RefitRPCHits = cms.bool(False),
+      Propagator = cms.string('SmartPropagatorAnyRKOpposite')
+  ),
+  minBX = cms.int32(-2),
+  maxBX = cms.int32(2)
 )
```

Comment:
If the one below are the only parameters modified with respect to the previous default, then if you restore them in the fillDescription method this config can be modified as simple as, without needing to touch the other configs
```suggestion
process.rpcPointProducer.ExtrapolatedRegion = 0.6, # in stripl/2 in Y and stripw*nstrips/2 in X
process.rpcPointProducer.cscSegments = ('dTandCSCSegmentsinTracks','SelectedCscSegments','OwnParticles'),
process.rpcPointProducer.dt4DSegments = ('dTandCSCSegmentsinTracks','SelectedDtSegments','OwnParticles'),
```

Reply:
Then you mean as long as the parameters are implemented in the fillDescription method already. Then I modify the [RecoLocalMuon/RPCRecHit/test/testRPCPointProducer.py] to include only the changed ones as you did above?